flocker_bb/builders/flocker.py

```diff
@@ -419,19 +407,23 @@ def makeOmnibusFactory(distribution, triggerSchedulers=()):
         description=['building', 'package'],
         descriptionDone=['build', 'package'],
         haltOnFailure=True))
-    factory.addSteps(createRepository(distribution))
+
+    repository_path = resultPath('omnibus', descriminator=distribution)
+
     factory.addStep(DirectoryUpload(
-        Interpolate('repo'),
-        Interpolate(b"private_html/omnibus/%s/%s" % (branch, distribution)),
-        url=Interpolate(
-            b"/results/omnibus/%s/%s/" % (branch, distribution),
-            ),
+        'repo',
+        repository_path,
+        url=resultURL('omnibus', descriminator=distribution),
         name="upload-repo",
-        ))
+    ))
+    factory.addSteps(createRepository(distribution, repository_path))
     if triggerSchedulers:
         factory.addStep(Trigger(
             name='trigger/built-rpms',
             schedulerNames=triggerSchedulers,
+            set_properties={
+                'merge_target': Property('lint_revision')
```

Comment:
Is `lint_revision` the revision against which the pyflakes builder is run? 
And is that chosen because it's likely to be the fastest builder?
Maybe add some explanatory comment.